src/translations/en.json

```diff
@@ -60,5 +60,7 @@
   "edit": "Edit",
   "like": "Like",
   "comment": "Comment",
-  "by": "by"
+  "by": "by",
+  "page_forbidden": "Page Requires Authentication",
```

Comment:
You have to translate on every language file overwise it would throw an error on the console and show the key on the UI.